mscore/instrwidget.cpp

```diff
@@ -88,20 +88,20 @@ void StaffListItem::initStaffTypeCombo(bool forceRecreate)
       // or in Instruments Wizard
       if (part) {
             const StringData* stringData = part->it ? &(part->it->stringData) :
-                        ( (part->part && part->part->instrument()) ? part->part->instrument()->stringData() : 0);
+                                                      ( (part->part && part->part->instrument()) ? part->part->instrument()->stringData() : 0);
```

Comment:
Not sure how large indents are usually used in this case but this change (and other similar changes) is anyway unrelated to this commit so it would be better not to include it here.